config/routes.rb

```diff
@@ -30,7 +30,8 @@
 
   # The work wizard
   get "works/new-submission", to: "works_wizard#new_submission", as: :work_create_new_submission
-  post "works/new-submission", to: "works_wizard#new_submission_save", as: :work_new_submission
+  post "works/new-submission/(:id)", to: "works_wizard#new_submission_save", as: :work_new_submission
```

Comment:
Curious, what do the parenthesis do?

Reply:
Patch is what is sent for an update, and (:id) allows the id to be present or not

Reply:
@hectorcorrea This allows for the save to occur for a brand new work, or one that was created because they hit save and returned to the page.